Extensions/3D/JsExtension.js

```diff
@@ -902,6 +903,12 @@ module.exports = {
         .setLabel(_('Depth'))
         .setMeasurementUnit(gd.MeasurementUnit.getPixel())
         .setGroup(_('Default size'));
+      objectProperties
+        .getOrCreate('cubeColor')
+        .setValue(objectContent.color || (255, 255, 255))
```

Comment:
Check for similar attributes (such as `fillColor` in the text input), how it's done